### DISCUSSION TITLE
accuracy assessment is not detecting classified images

### DISCUSSION THREAD

**Discussion by farhan7861991** [semiautomaticgit/SemiAutomaticClassificationPlugin#274] [Q&A]
hello
I am trying to use SCP for accuracy assessment but after i have imported classified file from arcgis the accuracy assessment tool and other tool in post processing is not recognizing the raster file so kindly tell me how i can rectify this issue. @@SemiAutomaticClassificationPlugin @semiautomaticgit Congedo @semiautomaticgit

**Comment by semiautomaticgit** [comment]
Hello, please copy the log file following these steps https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#how-can-i-report-an-error

**Reply by ivaninnocent** [reply]
i had the same issue. turns out  SCP version 7 has this issue. I had to get SCP 6 and it worked.  Generally with images classified in ARCGIS, when you transfer them to  SCP, its a nightmare analysing them. Symbology dissapears and change detection and accuracy assessment is a struggle. Processing errors occur and you may not get output in SCP but will get the results in the csv files.
